Week3/Homework/Js-exercises/totalCost.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+let calculateTotalPrice = function(cart) {
+  for (const value in cart) {
+    sum = sum + cart[value];
+  }
+
+  console.log(sum);
+};
+
+let sum = 0;
```

Comment:
Not a good idea to create the sum variable outside the function and then using it inside it. Better to have this inside the function. 🙂